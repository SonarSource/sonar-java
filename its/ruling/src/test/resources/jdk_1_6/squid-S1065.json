{
'jdk_1_6:java/io/BufferedReader.java':[
295,
],
}
