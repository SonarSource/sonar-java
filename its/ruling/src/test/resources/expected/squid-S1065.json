{
'project:com/sun/jmx/snmp/IPAcl/ParserTokenManager.java':[
769,
1121,
1330,
],
'project:java/io/BufferedReader.java':[
313,
],
'project:javax/management/ObjectName.java':[
566,
],
'project:javax/management/remote/JMXServiceURL.java':[
427,
],
}
